src/amidiplug/i_fileinfo.cc

```diff
@@ -348,6 +349,7 @@ bool i_fileinfo_gui (const char * filename_uri, VFSFile & file)
 
     title = g_path_get_basename (filename_utf8);
     gtk_window_set_title (GTK_WINDOW (fileinfowin), title);
+    gtk_window_set_role (GTK_WINDOW (fileinfowin), "fileinfo");
```

Comment:
It looks like `gtk_window_set_role()` is called twice for the same window (here and line 155).